frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/print/IPrintElementGenerator.java

```diff
@@ -0,0 +1,10 @@
+package com.virtuslab.gitmachete.frontend.graph.print;
```

Comment:
`print.elements`?

Reply:
I hope it all will be easier to understand soon